src/libs/actions/IOU.js

```diff
@@ -2302,6 +2302,8 @@ function navigateToNextPage(iou, iouType, report, path = '') {
     Navigation.navigate(ROUTES.MONEY_REQUEST_PARTICIPANTS.getRoute(iouType));
 }
 
+function submitReport() {}
```

Comment:
Should we add a comment like TODO, or some clarifications why we have an empty function?